### PR TITLE
fix(db/creature): Replace Smart Script AI on Verog the Dervish so that he paths down h…

### PR DESCRIPTION
**Pull request by DavuKnight**
## Changes Proposed:
-  Verog the Dervish doesnt move when he spawns this is not as per videos available of this quest. He is supposed to RP walk down the hill. I think this lack of movement is contributing to the impression that he doesnt spawn. This PR makes him walk down the hill.

## Issues Addressed:
- Closes 
https://github.com/chromiecraft/chromiecraft/issues/114
and
https://github.com/azerothcore/azerothcore-wotlk/issues/11236

## SOURCE:
https://classic.wowhead.com/npc=3395/verog-the-dervish#comments Lots of peeps from 2006 and 2007 qqing that it takes lots of kills (30+) to get a spawn even as rescent as 2020.

## Tests Performed:
- I killed for about 30 minutes with only one spawn but was to far away to get to him in 60 seconds.
- I did finally get him to spawn while being close enough to see. He does spawn. 
- Vorag now spawns. Spends 25 secs walking down the Hill and 35 seconds waiting before he despawns. This is the same 60 seconds hes always done just with the missing RP walk added. 

## How to Test the Changes:
1. .go xyz -1208.59	-2776.1	93.1576
2. Start killing everything in site for 30 to 45 minutes.
3. When something yells
4. .go xyz -1208.59	-2776.1	93.1576
5. verify Vorag is walking toward you and doesn't disappear when he reaches the bottum of the ramp for at least 30 seconds. 

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

**Review by Nefertumm — APPROVED** [review]

File: data/sql/updates/pending_db_world/rev_1661907895772938100.sql [hunk 1]
```diff
@@ -0,0 +1,8 @@
+-- Redoes teh SmartAI for Verog the Dervish so that he walks down the hillside after spawning
+UPDATE `creature_template` SET `AIName` = 'SmartAI' WHERE `entry` = 3395;
```
Comment (acidmanifesto) [inline data/sql/updates/pending_db_world/rev_1661907895772938100.sql:2]: he already has SmartAI linked in his smart script currently.

**Review by acidmanifesto — CHANGES_REQUESTED** [review]

**Review by acidmanifesto — APPROVED** [review]

**Comment by Gultask** [comment]
Tested and it works. Verog is still responsive when aggroed.

1. `.go xyz -1209.65 -2738.38 102.646 1 4.99352`
2. `.add npc temp 3395`

@DavuKnight what was your source for coords you provided? One screenshot in the Wowhead page shows him stationary at the spawn point

**Comment by DavuKnight** [comment]
So it does look like I have the coords wrong on the RP walk. If you frame by frame this one
https://www.youtube.com/watch?v=QEfs-_YYWuM

You can see he's slowly moving down the ramp. Which Is what I remember him doing. 

At the moment in AC however he's spawning at the wrong spot and walking to the wrong spot. He should be walking down the wood ramp not walking down the hill (the way I have him).

If you frame by frame that same video after the map pops you'll note that he is standing still (were A.C. has him spawning). This would agree with other videos where he appear to be standing still at the bottum of the ramp while yet others show that you camp him at the top of the ramp.

**Comment by Gultask** [comment]
Ok. No problem! I'll just add a 'Needs Sniff' tag and we can update the PR when someone gets one :)

**Review by acidmanifesto — CHANGES_REQUESTED** [review]
still awaiting sniffs or other credible source to confirm this.

**Comment by FrancescoBorzi** [comment]
what's the status of this PR?

**Comment by Nefertumm** [comment]
Sniffs needed but if we can't get them and the coords are pretty similar, we could merge and fix them later once we have the correct ones.

**Comment by Nyeriah** [comment]
The issue with merging stuff without sniffs is that nobody will come back to this to fix it later, because it will appear to be working even though its hackfixed/incorrect

**Comment by elthehablo** [comment]
> The issue with merging stuff without sniffs is that nobody will come back to this to fix it later, because it will appear to be working even though its hackfixed/incorrect

I did that. Sorry DavuKnight if you are still around, but I had to fix this

**Pull request closed without merging**